main/models.py

```diff
@@ -252,9 +262,9 @@ def payments_to_cashflows(date, payments):
         if p.date > date:
             continue
         if p.description:
-            d = _('payment "%(description)s"') % {'description': p.description}
+            d = _('PAYMENT "%(description)s"') % {'description': p.description}
```

Comment:
Ok. We can keep uppercase

On 18 November 2015 at 13:35, vectorien <notifications@github.com> wrote:

> In main/models.py
> <https://github.com/oadam/proprio/pull/21#discussion_r45193474>:
>
> > @@ -252,9 +262,9 @@ def payments_to_cashflows(date, payments):
> >          if p.date > date:
> >              continue
> >          if p.description:
> > -            d = _('payment "%(description)s"') % {'description': p.description}
> > +            d = _('PAYMENT "%(description)s"') % {'description': p.description}
>
> hm i thought it stands out better if it is all uppercase..
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oadam/proprio/pull/21/files#r45193474>.
>